config/envs/default.py

```diff
@@ -30,6 +30,10 @@ class DefaultConfig:
         "INCOMPLETE_APPLICATION_TEMPLATE_ID",
         "fc8cff7c-a595-4590-a1a4-eccda48d8604",
     )
+    APPLICATION_DEADLINE_REMINDER_TEMPLATE_ID = os.environ.get(
+        "APPLICATION_DEADLINE_REMINDER_TEMPLATE_ID",
+        "e41cc73d-6947-4cbb-aedd-4ab2f470a2d2",
+    )
```

Comment:
⛏️ Is it safe to have this be the default? Surely this should fail if an template id hasnt been given?

Reply:
Yes, but I was told to keep it in config file as it is just the template id & it will be in cf as well. I think the api key in important?